README.adoc

```diff
@@ -163,6 +163,7 @@ spring.cloud.vault:
     scheme: http
     connection-timeout: 5000
     read-timeout: 15000
+    token: some-application-token
```

Comment:
Makes sense but without any authentication schema specified an application using this extension exits stating that an authentication token is mandatory.  I didn't see any mention of other authentication schemes in the docs (only in the code) and added as a immediate assist for new comers (like myself :-)) most likely to use the extension with Vaults primary authentication scheme.  Agreed, this is probably best broken out into a separate section on authentication.